src/components/kepler-gl.js

```diff
@@ -153,6 +154,9 @@ function KeplerGlFactory(
     }
 
     _loadMapStyle = () => {
+      if(this.props.mapStylesReplaceDefault) {
```

Comment:
Is the intention here to discard all default map styles provided by kepler? Not even show them in the map style selection panel? If that's the case, there should be a cleaner way to do this. See comments in `map-style-updaters.js`

Reply:
🍏 Done